### PR TITLE
refactor: move private test helpers to utils module to be used by many modules

### PR DESCRIPTION
**Pull request by NGA-TRAN**
Closes #

Make private test helpers public to be more broadly used

- [ ] I've read the contributing section of the project [README](https://github.com/influxdata/influxdb_iox/blob/main/README.md).
- [ ] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed).

### CODE REVIEW

File: parquet_file/src/lib.rs [hunk 1]
```diff
@@ -12,3 +12,4 @@ pub mod chunk;
 pub mod metadata;
 pub mod storage;
 pub mod table;
+pub mod utils;
```
Comment (domodwyer) [inline parquet_file/src/lib.rs:15]: If these helpers should only be used for tests, I usually mark them as test only to keep them out of production code:

```suggestion
#[cfg(test)]
pub mod utils;
```

This means they can only be imported when running tests - it depends how you want them to be used though!

**Review by domodwyer — APPROVED** [review]

**Review by e-dard — APPROVED** [review]
I think this is fine. Do we only want to use this code for tests though? I wonder if we could use cargo features to hide it all behind a feature, and then turn that feature on in `dev-dependencies`? Not sure what others think

File: parquet_file/src/lib.rs (hunk 1 above)
Comment (NGA-TRAN) [reply parquet_file/src/lib.rs:15]: Yes, keep it test only for now

**Comment by NGA-TRAN** [comment]
@e-dard Currenty this is only used for test only ad I will follow @domodwyer 's suggestion

**Pull request merged**